app/src/main/java/org/oppia/android/app/player/state/StateFragment.kt

```diff
@@ -129,6 +137,11 @@ class StateFragment :
     stateFragmentPresenter.revealHint(hintIndex)
   }
 
+  override fun onSaveInstanceState(outState: Bundle) {
+    super.onSaveInstanceState(outState)
+    outState.putProto("Answer", stateFragmentPresenter.handleOnSavedInstance())
```

Comment:
Here & above: please add a proto key for the saved instance state like we do elsewhere in the codebase.